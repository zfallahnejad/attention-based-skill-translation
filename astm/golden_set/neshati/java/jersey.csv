2587435,jersey
950199,jersey
383861,jersey
290799,jersey
680925,jersey
463324,jersey
59501,jersey
1596371,jersey
438154,jersey
945485,jersey
547779,jersey
