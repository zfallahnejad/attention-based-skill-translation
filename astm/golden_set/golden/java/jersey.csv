2587435,71,29,0.71
950199,54,34,0.6136363636363636
383861,44,36,0.55
290799,38,19,0.6666666666666666
680925,26,8,0.7647058823529411
463324,22,20,0.5238095238095238
59501,15,18,0.45454545454545453
1596371,15,12,0.5555555555555556
945485,12,9,0.5714285714285714
438154,12,3,0.8
547779,11,8,0.5789473684210527
